app_dart/lib/src/service/github_service.dart

```diff
@@ -15,27 +15,40 @@ class GithubService {
   final GitHub github;
 
   /// Lists the commits of the provided repository [slug] and [branch].
```

Comment:
Missing documentation about how this function behaves against the `lastCommitTimestampMills` parameter.

Reply:
Added.